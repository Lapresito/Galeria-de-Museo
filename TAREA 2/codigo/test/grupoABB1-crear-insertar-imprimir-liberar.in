crearGrupoABB
crearVisitante 4 Ubaldo Rodriguez 25
agregarAGrupoABB
crearVisitante 10 Artemio Martinez 20
agregarAGrupoABB
crearVisitante 5 Brunilda Garcia 14
agregarAGrupoABB
crearVisitante 1 Ubaldo Gonzalez 57
agregarAGrupoABB
imprimirGrupoABB
crearVisitante 11 Ubaldo Martinez 54
agregarAGrupoABB
crearVisitante 14 Zenobia Fernandez 20
agregarAGrupoABB
crearVisitante 17 Artemio Fernandez 20
agregarAGrupoABB
crearVisitante 12 Esmeralda Martinez 20
agregarAGrupoABB
imprimirGrupoABB
crearVisitante 2 Ubaldo Gomez 24
agregarAGrupoABB
crearVisitante 3 Zenobia Martinez 23
agregarAGrupoABB
imprimirGrupoABB
liberarGrupoABB
Fin
