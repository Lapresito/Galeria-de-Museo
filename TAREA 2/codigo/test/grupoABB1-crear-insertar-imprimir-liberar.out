Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El grupoABB ha sido creado de forma exitosa.
2> Visitante creadox de forma exitosa.
3> Se ha agregado el visitante al grupo de forma exitosa.
4> Visitante creadox de forma exitosa.
5> Se ha agregado el visitante al grupo de forma exitosa.
6> Visitante creadox de forma exitosa.
7> Se ha agregado el visitante al grupo de forma exitosa.
8> Visitante creadox de forma exitosa.
9> Se ha agregado el visitante al grupo de forma exitosa.
10> Visitante Ubaldo Gonzalez
Id: 1
Edad: 57
Visitante Ubaldo Rodriguez
Id: 4
Edad: 25
Visitante Brunilda Garcia
Id: 5
Edad: 14
Visitante Artemio Martinez
Id: 10
Edad: 20
11> Visitante creadox de forma exitosa.
12> Se ha agregado el visitante al grupo de forma exitosa.
13> Visitante creadox de forma exitosa.
14> Se ha agregado el visitante al grupo de forma exitosa.
15> Visitante creadox de forma exitosa.
16> Se ha agregado el visitante al grupo de forma exitosa.
17> Visitante creadox de forma exitosa.
18> Se ha agregado el visitante al grupo de forma exitosa.
19> Visitante Ubaldo Gonzalez
Id: 1
Edad: 57
Visitante Ubaldo Rodriguez
Id: 4
Edad: 25
Visitante Brunilda Garcia
Id: 5
Edad: 14
Visitante Artemio Martinez
Id: 10
Edad: 20
Visitante Ubaldo Martinez
Id: 11
Edad: 54
Visitante Esmeralda Martinez
Id: 12
Edad: 20
Visitante Zenobia Fernandez
Id: 14
Edad: 20
Visitante Artemio Fernandez
Id: 17
Edad: 20
20> Visitante creadox de forma exitosa.
21> Se ha agregado el visitante al grupo de forma exitosa.
22> Visitante creadox de forma exitosa.
23> Se ha agregado el visitante al grupo de forma exitosa.
24> Visitante Ubaldo Gonzalez
Id: 1
Edad: 57
Visitante Ubaldo Gomez
Id: 2
Edad: 24
Visitante Zenobia Martinez
Id: 3
Edad: 23
Visitante Ubaldo Rodriguez
Id: 4
Edad: 25
Visitante Brunilda Garcia
Id: 5
Edad: 14
Visitante Artemio Martinez
Id: 10
Edad: 20
Visitante Ubaldo Martinez
Id: 11
Edad: 54
Visitante Esmeralda Martinez
Id: 12
Edad: 20
Visitante Zenobia Fernandez
Id: 14
Edad: 20
Visitante Artemio Fernandez
Id: 17
Edad: 20
25> Grupo liberado con exito.
26> Fin.
